src/store/message-store-level.ts

```diff
@@ -244,11 +158,15 @@ export class MessageStoreLevel implements MessageStore {
   async put(
     tenant: string,
     message: GenericMessage,
-    indexes: { [key: string]: string | boolean },
+    indexes: KeyValues,
     options?: MessageStoreOptions
   ): Promise<void> {
     options?.signal?.throwIfAborted();
 
+    if (indexes.messageTimestamp === undefined) {
```

Comment:
For my understanding: why such a check? Doesn't every message have `messageTimestamp` already?

Reply:
Yes, and I don't mind removing it if you think we don't need it here.

I thought it would be a good check since `MessageStore` is it's own interface, but it's currently checked in the various schema validators.

Reply:
I guess I am trying to understand why `messageTimestamp` is more special that we are making explicit check for it? I mean why don't we check for say `interface`, `method`, `author` etc?

If no more special, then we can just remove, else you'll need to write test for it (or you already have!)